docs/vm/live-migration.md

```diff
@@ -56,3 +56,22 @@ Live migration will be aborted if it exceeds the completion timeout of 800s per
 ### Progress Timeout
 
 Live migration will also be aborted when copying memory doesn't make any progress in 150s.
+
+## Limitations
+
+When performing live migration, the process will check the value of `spec.domain.cpu.model` in VMI CR, which is derived from `spec.template.spec.domain.cpu.model` in VM CR. If this vaule is not set, it'll defaults to `host-model`.
+
+With `host-model`, it only allows the live migration process to migrate the VM to a node with same CPU model. For example, if the original CPU model is `XYZ`, it can migrate VM to another node with `XZY` CPU model. However, sometimes VM does not require a specified CPU model. In such case, we need to shut down the VM, assign a CPU model supported by all nodes, and then restart the VM.
+
+Each node has primary CPU model labeled with `host-model-cpu.node.kubevirt.io` and mutliple supported CPU models which are labeled with `cpu-model.node.kubevirt.io`.
```

Comment:
we should probably move this higher up in the section to explain how users can check `host-model` before they even launch a VM to help them decide if they wish to pin a different model?

Reply:
@ibrokethecloud I've pushed new version.

Reply:
I added the sentence to paragraph 1. Please remove this line.